src/pages/python/adding-alerts-and-slos.mdx

```diff
@@ -0,0 +1,57 @@
+import { Callout, Steps, Tabs, Tab } from "nextra-theme-docs";
+import AlertsSLOsLangIntro from "@/snippets/adding-alerts-slos.mdx";
+
+<AlertsSLOsLangIntro language="Python"/>
+
+## Pre-requisites
+
+1. Make sure you have setup and initialized Autometrics libraries as described in [Quickstart section](quickstart).
```

Comment:
nits: "set up" not "setup", and "in **the** Quickstart section" (also not sure if Quickstart needs capitalization)